_episodes/05-starting-jekyll.md

```diff
@@ -7,75 +7,351 @@ questions:
-> Some possible options include: your name, the institution you work at, the PI of your lab group.
-> Create the variable and reference it in your `index.md` and save/commit that to your repository.
-> What did you add to your `index.md` to create this variable?
-> Where did you add the line above to your `index.md`?
-> How did you reference that variable?
+> [YAML](https://yaml.org/) is a human-readable data-serialization language. It is commonly used for configuration files and in
+> applications where text data is being stored or transmitted and it is programming language agnostic.
 >
+{: .callout}
+
+Let's create some configuration parameters for our website.
+
+1. From the GitHub interface, create `_config.yml` file in your site’s root directory.
+2. Add parameters `title`, `description` and `email` to it as:
+
+    ~~~
+    title: "Building Websites in GitHub"
+    description: "This research project develops training materials for reseachers wanting to learn to build project
+    websites in GitHub with GitHub Pages."
+    email: "team@my.research.org"
+    ~~~
+    {: .language-yaml}
+
+3. Commit your changes.
+
+Global configuration settings from
+`_config.yml` are made available as `site.PARAMETER_NAME` variable in every page within the website. So, global parameter `email` we defined above would be accessed as `site.email`. 
+
+In order to access the parameter's value within a page, you use Liquid's notation to output content by surrounding a variable in curly braces as `{% raw %}{{ variable }}{% endraw %}`. 
+
+> ## Predefined Global Parameters
+>In addition to the global parameters you define, Jekyll also makes a number of
+[useful predefined site-wide variables](https://jekyllrb.com/docs/variables#site-variables) available to you within your website: e.g. `{% raw %}{{ site.time }}{% endraw %}` (the current time) or `{% raw %}{{ site.pages }}{% endraw %}` (a list of all pages).
+{: .callout}
+
+Let's make use of global parameters in our pages.
+
+1. Modify `index.md` file to make use of our global parameters like this:
+
+   ~~~
+    # {% raw %}{{ site.title }}{% endraw %}
+   
+    ## Description
+    {% raw %}{{ site.description }}{% endraw %}
+
+    Have any questions about what we do? [We'd love to hear from you!]({% raw %}mailto:{{ site.email }}{% endraw %})
+   ~~~  
+   
+2. We can use the same parameter in different pages. Let's reuse `{% raw %}{{ site.description }}{% endraw %}` and 
+`{% raw %}{{ site.email }}{% endraw %}` in `about.md` like this:
+
+    ~~~ 
+    # About
+   
+    ## Project
+    {% raw %}{{ site.description }}{% endraw %}
+           
+    ## Funders
+    We gratefully acknowledge funding from the XYZ Founding Council, under grant number 'abc'.
+   
+    ## Cite us
+    You can cite the project as:  
+    
+    >    *My research project. Zenodo. https://zenodo.org/record/doi*
+    
+    ## Contact us
+   
+    Email: [team@my.research.org](mailto:{% raw %}{{ site.email }}{% endraw %})   
```

Comment:
think maybe the link should be   
```suggestion
    [Email](mailto:{% raw %}{{ site.email }}{% endraw %})
```
otherwise the link text shown doesn't make use of the variable which works against the purpose of variable a little.  Alternative maybe the following works?
```suggestion
    Email: [{% raw %}{{ site.email }}{% endraw %}](mailto:{% raw %}{{ site.email }}{% endraw %})   
```